packages/create-mud/package.json

```diff
@@ -16,6 +16,7 @@
     "dev": "tsup --watch",
     "prepublishOnly": "npm run clean && npm run build",
     "test": "pnpm run test:vanilla && pnpm run test:react",
+    "test:phaser": "dist/cli.js test-project --template phaser && rimraf test-project",
```

Comment:
you'll wanna edit the CI step to include phaser too: https://github.com/latticexyz/mud/blob/main/.github/workflows/templates.yml#L15